indico/modules/events/management/controllers.py

```diff
@@ -126,6 +127,33 @@ def _process(self):
                                 event_persons=contribution_persons_dict, event=self.event_new)
 
 
+class RHAbstractPersonListMixin:
```

Comment:
Why a mixin? It's only used in one place unlike the one for sessions/contributions (which is why that one was a mixin)